Makefile

```diff
@@ -23,34 +33,28 @@ endif
 # bingo manages consistent tooling versions for things like kind, kustomize, etc.
 include .bingo/Variables.mk
 
-# ARTIFACT_PATH is the absolute path to the directory where the operator-controller e2e tests will store the artifacts
-# for example: ARTIFACT_PATH=/tmp/artifacts make test
```

Comment:
losing the useless export seems good, but do we want to keep this bit of dev knowledge around somehow?

Reply:
There's nothing in the Makefile that uses it.

Just this: https://github.com/operator-framework/operator-controller/blob/f22ce65c3cfd7f548709a4730f5f83095847ff8b/.github/workflows/e2e.yaml#L31
And this: https://github.com/operator-framework/operator-controller/blob/f22ce65c3cfd7f548709a4730f5f83095847ff8b/test/e2e/cluster_extension_install_test.go#L380

Maybe we could just leave this comment in the Makefile next to the `e2e` target though?